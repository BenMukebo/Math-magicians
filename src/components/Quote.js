import React from 'react';

const Quote = () => (
  <section className="">
    <p className="">
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Non ad quod
      accusamus nobis maxime facere possimus earum officia rem aliquid.
      Voluptatibus in provident ducimus dignissimos illo, temporibus reprehenderit quis minima.
    </p>
  </section>
);

export default Quote;
