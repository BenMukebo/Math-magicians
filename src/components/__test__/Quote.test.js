import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

it('matches snapshot', () => {
  const tree = renderer.create(
    <Quote />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
