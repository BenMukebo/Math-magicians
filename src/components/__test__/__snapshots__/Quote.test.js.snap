// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<section
  className="quote-section"
>
  <p
    className=""
  >
    Without calculators, advanced math courses, such as Calculus, would require much longer class-times and reformatted lesson plans. Calculators, when used alongside a comprehensive mathematics curriculum, increase the quality and quantity of student learning.
  </p>
</section>
`;
